cmd/treedb_text_ingest_qual/produce.go

```
@@ -0,0 +1,301 @@
+	}
+	return nil
+}
+
+// produceOneMode is called only by the fresh child. Its DB is deliberately
+// outside the raw-row directory and always removed after the row is copied.
+func produceOneMode(dir, mode string, scale, repetition int) error {
+	dbDir, err := os.MkdirTemp("", "gomap-4328-text-ingest-"+mode+"-")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(dbDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)